src/java.base/share/classes/java/util/SequencedMap.java

```diff
@@ -340,44 +340,4 @@ public SequencedSet<Map.Entry<K, V>> reversed() {
         }
         return new SeqEntrySet();
     }
-
-    /**
-     * Delegates all Collection methods to the provided non-sequenced map view,
-     * except add() and addAll(), which throw UOE. This provides the common
-     * implementation of each of the sequenced views of the SequencedMap.
-     * Each view implementation is a subclass that provides an intance of the
-     * non-sequenced view as a delegate and an implementation of the reversed().
-     * Each view also inherits the default implementations for the sequenced
-     * methods from SequencedCollection or SequencedSet.
-     *
-     * @param <E> the view's element type
-     */
-    class AbstractViewCollection<E> implements Collection<E> {
-        UnsupportedOperationException uoe() { return new UnsupportedOperationException(); }
-        final Collection<E> view;
-
-        AbstractViewCollection(Collection<E> view) { this.view = view; }
-
-        public boolean add(E t) { throw uoe(); }
-        public boolean addAll(Collection<? extends E> c) { throw uoe(); }
-        public void clear() { view.clear(); }
-        public boolean contains(Object o) { return view.contains(o); }
-        public boolean containsAll(Collection<?> c) { return view.containsAll(c); }
-        public boolean equals(Object o) { return view.equals(o); }
-        public void forEach(Consumer<? super E> c) { view.forEach(c); }
-        public int hashCode() { return view.hashCode(); }
-        public boolean isEmpty() { return view.isEmpty(); }
-        public Iterator<E> iterator() { return view.iterator(); }
-        public Stream<E> parallelStream() { return view.parallelStream(); }
-        public boolean remove(Object o) { return view.remove(o); }
-        public boolean removeAll(Collection<?> c) { return view.removeAll(c); }
-        public boolean removeIf(Predicate<? super E> filter) { return view.removeIf(filter); }
-        public boolean retainAll(Collection<?> c) { return view.retainAll(c); }
-        public int size() { return view.size(); }
-        public Spliterator<E> spliterator() { return view.spliterator(); }
-        public Stream<E> stream() { return view.stream(); }
-        public Object[] toArray() { return view.toArray(); }
-        public <T> T[] toArray(IntFunction<T[]> generator) { return view.toArray(generator); }
-        public <T> T[] toArray(T[] a) { return view.toArray(a); }
-    }
-}
+}
```

Comment:
Missing trailing newline:
```suggestion
}
```

Reply:
will fix